parsec/core/types/base.py

```diff
@@ -4,11 +4,12 @@
 
 from parsec.serde import BaseSchema, MsgpackSerializer
 from parsec.api.data import BaseData, EntryName
-
+import attr
```

Comment:
3rd party imports should be above parsec imports

Reply:
that's why we need isort ! fixed